cluster-operator/src/test/java/io/strimzi/operator/cluster/FeatureGatesTest.java

```diff
@@ -64,12 +64,12 @@ public void testFeatureGatesParsing() {
         assertThat(new FeatureGates("  -UseKRaft    ").useKRaftEnabled(), is(false));
         // TODO: Add more tests with various feature gate combinations once we have multiple feature gates again.
         //       The commented out code below shows the tests we used to have with multiple feature gates.
-        //assertThat(new FeatureGates("-UseKRaft,-KafkaNodePools").useKRaftEnabled(), is(false));
-        //assertThat(new FeatureGates("-UseKRaft,-KafkaNodePools").kafkaNodePoolsEnabled(), is(false));
-        //assertThat(new FeatureGates("  +UseKRaft    ,    +KafkaNodePools").useKRaftEnabled(), is(true));
-        //assertThat(new FeatureGates("  -UseKRaft    ,    -KafkaNodePools").kafkaNodePoolsEnabled(), is(false));
-        //assertThat(new FeatureGates("+KafkaNodePools,-UseKRaft").useKRaftEnabled(), is(false));
-        //assertThat(new FeatureGates("+KafkaNodePools,-UseKRaft").kafkaNodePoolsEnabled(), is(true));
+        assertThat(new FeatureGates("-UseKRaft,-ContinueReconciliationOnManualRollingUpdateFailure").useKRaftEnabled(), is(false));
+        assertThat(new FeatureGates("-UseKRaft,-ContinueReconciliationOnManualRollingUpdateFailure").continueReconciliationOnManualRollingUpdateFailureEnabled(), is(false));
+        assertThat(new FeatureGates("+UseKRaft,+ContinueReconciliationOnManualRollingUpdateFailure").useKRaftEnabled(), is(true));
+        assertThat(new FeatureGates("+UseKRaft,+ContinueReconciliationOnManualRollingUpdateFailure").continueReconciliationOnManualRollingUpdateFailureEnabled(), is(true));
```

Comment:
These tests are good -> but can we keep the original whitespaces? That is the part they were supposed to test.